src/app/ApplicationTemplate.Shared.Views/Styles/Controls/Button.xaml

```diff
@@ -1,9 +1,29 @@
-﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 					xmlns:u="using:Nventive.View.Controls">
 
+	<!-- Override FilledButtonStyle to avoid rounding buttons -->
+	<Style	x:Key="FilledButtonStyle"
+			TargetType="Button"
+			BasedOn="{StaticResource FilledButtonStyle}">
+		<Setter Property="CornerRadius"
+				Value="4" />
+		<Setter Property="FontFamily"
+				Value="{StaticResource FontFamily01}" />
```

Comment:
Why do we override the fonts here and not via the fonts override?

Reply:
removed, it's now using the overwritten file for fonts